apps/api/src/app/app.service.ts

```diff
@@ -12,27 +13,33 @@ export class AppService {
     private readonly redisService: RedisService,
     private prisma: PrismaService,
     private telemetryService: TelemetryService,
+    private redisUtils: RedisUtils,
     ) {}
 
-  async setKey(hashid: string): Promise<void> {
-    const client = await this.redisService.getClient(this.configService.get<string>('REDIS_NAME'));
-    client.set(hashid, 0);
-  }
-  
-  async updateClicks(urlId: string): Promise<void> {
-    const client = await this.redisService.getClient(this.configService.get<string>('REDIS_NAME'));
-    client.incr(urlId);
-  }
 
-  async fetchAllKeys(): Promise<string[]> {
-    const client = await this.redisService.getClient(this.configService.get<string>('REDIS_NAME'));
-    const keys: string[] = await client.keys('*');
-    return keys
-  }
+    /**
+     * Updates the click count in the postgres db based on hashId or customhashid
+     * @param hashID 
+     */
+    async updateClicksInPostgresDB(hashID: string): Promise<void> {
```

Comment:
it should just be `id` bc we don't know now if it is hashid or customhashid.

Reply:
and type should be number | string

Reply:
@yuvrajsab 
Here I just need to change the name of parameter and it's type 
Right ?

Reply:
resolved in 30ba88d124a83df18c841cfeb29b93247d842f49